massgov/mayflower
# React/DP-14339:  EmergencyHeader Component.

## Description
- (React) [EmergencyHeader] DP-14339: Adds the EmergencyHeader component.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-14339)

## Steps to Test


1. Pull the branch, start up storybook. Browse to molecules -> EmergencyHeader and verify the functionality works as intended.

## Screenshots
<img width="1462" alt="Screen Shot 2019-06-13 at 5 01 51 PM" src="https://user-images.githubusercontent.com/5257660/59467161-f862a480-8dfc-11e9-829e-8359c9db9af4.png">
